packages/rn-tester/js/examples/Transform/TransformExample.js

```diff
@@ -50,6 +50,45 @@ function AnimateTransformSingleProp() {
   );
 }
 
+function AnimateTranslateOrigin({transformOrigin}: {transformOrigin: string}) {
+  const [theta] = useState(new Animated.Value(45));
+  const animate = () => {
+    theta.setValue(0);
+    Animated.timing(theta, {
+      toValue: 100,
+      duration: 3000,
+      useNativeDriver: false,
+    }).start(animate);
+  };
+
+  useEffect(() => {
+    animate();
+  });
+
+  return (
+    <View style={[styles.flipCardContainer]}>
+      <View style={{borderWidth: 1, alignSelf: 'center'}}>
+        <Animated.View
+          style={[
+            styles.translateOriginBox,
+            {
+              transformOrigin,
+              transform: [
+                {
+                  rotate: theta.interpolate({
+                    inputRange: [0, 100],
+                    outputRange: ['0deg', '360deg'],
+                  }),
+                },
+              ],
```

Comment:
To validate this works even when transform doesn't change, wrap the transform value in a `useMemo` so it doesn't change across renders.